pkg/scheduler/core/assignment_test.go

```diff
@@ -591,8 +591,11 @@ func Test_dynamicScaleUp(t *testing.T) {
 		candidates []*clusterv1alpha1.Cluster
 		object     *workv1alpha2.ResourceBindingSpec
 		placement  *policyv1alpha1.Placement
-		want       []workv1alpha2.TargetCluster
-		wantErr    bool
+		// want specifies exactly one detail desired result (lose effect when wants defined)
+		want []workv1alpha2.TargetCluster
+		// wants specifies multi possible desired result, any one got is expected
+		wants   [][]workv1alpha2.TargetCluster
```

Comment:
Can we deprecate the `want` and put all expectations into `wants`?

Reply:
This is also possible. I initially didn't want to affect too many lines